### ISSUE TITLE
need to be able to add, delete, edit code domain names in the coded field manager

### ISSUE THREAD

**Issue by difiore** [ethoinformatics/ethoinfo-project-app#13]

**Comment by relevante** [comment]
OK cool - this looks like it requires a semi-significant architecture change. At the moment these are specified at ethoinfo-project-app in the javascript that sets up the app (in what would eventually become the JSON configuration document).

**Comment by difiore** [comment]
ah, right... but folks could edit the javascript, right? so, if I wanted to get rid of the "Social behavior" domain for a workaround, I could edit in there, ya?

**Comment by difiore** [comment]
I see where in the index file this would be edited. not going to, of course :)

**Comment by relevante** [comment]
Yeah exactly - right now this is the kind of thing that someone could change in their specific app when setting it up, but not from within the app. If it needs to be more dynamic than that, we'll have to restructure the way it's set up.

**Comment by tigoe** [comment]
This feels like it's not something that goes in the app, it seems like development time stuff. @difiore can we call this one closed?
